src/pages/resources/explore-books/explore-books.ts

```diff
@@ -0,0 +1,198 @@
+import {Component} from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import {Map} from "@app/app";
+import {Environment, InteractSubtype, InteractType, PageId} from "@app/service/telemetry-constants";
+import {ContentSearchCriteria, ContentService, SearchType} from "sunbird-sdk/dist";
+import {AppHeaderService, CommonUtilService} from "@app/service";
+import { animate, group, state, style, transition, trigger } from '@angular/animations';
+
+
```

Comment:
Not a Single Telemetry Event is raised for the entire page. Which is a potential blocker